--- cherrystrap/dbMap.py	2015-02-11 15:59:58.246002010 +1100
+++ cherrystrap/dbMap.py	2015-02-11 15:57:03.268238311 +1100
@@ -17,8 +17,8 @@
 		table_list['albuminfo'] = music_database+'.album'
 
 	elif version == "Helix":
-		music_database = 'xbmc_music48'
-		video_database = 'xbmc_video90'
+		music_database = 'MyMusic48'
+		video_database = 'MyVideos90'
 
 		table_list['artistinfo'] = music_database+'.artist'
 		table_list['albuminfo'] = music_database+'.album'
@@ -45,4 +45,4 @@
 	table_list['musicvideo'] = video_database+'.musicvideo'
 	table_list['musicvideoview'] = video_database+'.musicvideoview'
 
-	return table_list
\ No newline at end of file
+	return table_list
--- cherrystrap/database.py	2015-02-11 15:59:58.245002017 +1100
+++ cherrystrap/database.py	2015-02-11 15:56:38.093416182 +1100
@@ -88,7 +88,7 @@
         port = cherrystrap.XBMC_PORT
         if port:
             try:
-                port = int(cherrystrap.VANILLA_PORT)
+                port = int(cherrystrap.XBMC_PORT)
             except:
                 logger.error("The port number supplied is not an integer")
         else:
