--- googleplaydownloader/__init__.py	2015-01-27 10:25:26.000000000 +1100
+++ googleplaydownloader/__init__.py	2015-03-22 13:31:33.273875259 +1100
@@ -4,7 +4,7 @@
 
 import os
 HERE = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(HERE, "googleplaydownloader", "version.txt"), "r") as f:
+with open(os.path.join(HERE, "version.txt"), "r") as f:
   __version__ = f.read().strip()
 
 from .googleplaydownloader import main as start_gui
--- googleplaydownloader/googleplaydownloader.py	2015-01-27 10:15:52.000000000 +1100
+++ googleplaydownloader/googleplaydownloader.py	2015-03-22 13:29:47.889119657 +1100
@@ -18,10 +18,10 @@
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 _icons_path = os.path.join(HERE, 'img')
-
-from ext_libs.googleplay_api.googleplay import GooglePlayAPI #GooglePlayAPI
-from ext_libs.googleplay_api.googleplay import LoginError
-from ext_libs.androguard.core.bytecodes import apk as androguard_apk #Androguard
+sys.path.insert(0, os.path.join(HERE,'./ext_libs'))
+from googleplay_api.googleplay import GooglePlayAPI #GooglePlayAPI
+from googleplay_api.googleplay import LoginError
+from androguard.core.bytecodes import apk as androguard_apk #Androguard
 
 
 config = {}
