--- googleplaydownloader/googleplaydownloader.py	2015-01-27 10:15:52.000000000 +1100
+++ googleplaydownloader/googleplaydownloader.py	2015-05-08 11:49:13.959996306 +1000
@@ -18,10 +18,10 @@
 
 HERE = os.path.abspath(os.path.dirname(__file__))
 _icons_path = os.path.join(HERE, 'img')
-
-from ext_libs.googleplay_api.googleplay import GooglePlayAPI #GooglePlayAPI
-from ext_libs.googleplay_api.googleplay import LoginError
-from ext_libs.androguard.core.bytecodes import apk as androguard_apk #Androguard
+sys.path.insert(0, os.path.join(HERE,'./ext_libs'))
+from googleplay_api.googleplay import GooglePlayAPI #GooglePlayAPI
+from googleplay_api.googleplay import LoginError
+from androguard.core.bytecodes import apk as androguard_apk #Androguard
 
 
 config = {}
@@ -119,6 +119,7 @@
 def analyse_local_apks(list_of_apks, playstore_api, download_folder_path, dlg, return_function):
   list_apks_to_update = []
   for position, filename in enumerate(list_of_apks):
+    print "%i/%i : %s\nPlease Wait..." %(position+1, len(list_of_apks), filename)
     wx.CallAfter(dlg.Update, position, "%i/%i : %s\nPlease Wait..." %(position+1, len(list_of_apks), filename))
 
     #Get APK info from file on disk
@@ -156,7 +157,7 @@
     doc = m.docV2
     title = doc.title
     vc = doc.details.appDetails.versionCode
-
+    print "%i/%i : %s\n%s\nSize : %s\nPlease Wait..." %(position+1, len(list_of_packages_to_download), title, packagename, sizeof_fmt(doc.details.appDetails.installationSize))
     #Update progress dialog
     wx.CallAfter(dlg.Update, position, "%i/%i : %s\n%s\nSize : %s\nPlease Wait...(there is no download progression)" %(position+1, len(list_of_packages_to_download), title, packagename, sizeof_fmt(doc.details.appDetails.installationSize)))
 
@@ -180,7 +181,6 @@
   wx.CallAfter(dlg.Update, position+1)   #Reach end of progress dialog
   wx.CallAfter(return_function,failed_downloads)
 
-
 def softwareID(query) :
   if query == "name":
     return u"Google Play Downloader"
@@ -282,26 +282,30 @@
     bigbox.Add(updatebox, 0, wx.EXPAND|wx.ADJUST_MINSIZE)
     bigbox.Add(creditsbox, 0)
 
-
     self.SetSizer(bigbox)
     self.SetMinSize((700,400))
     search_entry.SetFocus()
-
+    self.autoUpdate = False 
 
   def prepare_analyse_apks(self, event):
-    if self.ask_download_folder_path() == True:
+    if self.autoUpdate == False:
+      if self.ask_download_folder_path() == True:
+        download_folder_path = config["download_folder_path"]
+      else:
+        return True
+    else:
       download_folder_path = config["download_folder_path"]
-      list_of_apks = [filename for filename in os.listdir(download_folder_path) if os.path.splitext(filename)[1] == ".apk"]
-      if len(list_of_apks) > 0:
-        dlg = wx.ProgressDialog("Updating APKs",
+    
+    list_of_apks = [filename for filename in os.listdir(download_folder_path) if os.path.splitext(filename)[1] == ".apk"]
+    if len(list_of_apks) > 0:
+      print "Updating APKs"
+      dlg = wx.ProgressDialog("Updating APKs",
                                    "_" * 30 + "\n"*2,
                                    maximum = len(list_of_apks),
                                    parent=self,
                                    style = wx.PD_AUTO_HIDE
                                    )
-        thread.start_new_thread(analyse_local_apks, (list_of_apks, self.playstore_api, download_folder_path, dlg, self.prepare_download_updates))
-
-
+      thread.start_new_thread(analyse_local_apks, (list_of_apks, self.playstore_api, download_folder_path, dlg, self.prepare_download_updates))
 
 
   def search(self, results_list, search_string, nb_results):
@@ -405,11 +409,15 @@
         else:
           message += "\n%s" % package_name
         message += "\n%s\n" % exception
-
-    #Show info dialog
-    dlg = wx.MessageDialog(self, message,'Download report', wx.OK | wx.ICON_INFORMATION)
-    dlg.ShowModal()
-    dlg.Destroy()
+    print message
+    if self.autoUpdate == False:
+      #Show info dialog
+      dlg = wx.MessageDialog(self, message,'Download report', wx.OK | wx.ICON_INFORMATION)
+      dlg.ShowModal()
+      dlg.Destroy()
+    else:
+      self.autoUpdate = False
+      quit()
 
   def ask_download_folder_path(self):
     dlg = wx.DirDialog(self, "Choose a download folder:",  defaultPath = config["download_folder_path"], style=wx.DD_DEFAULT_STYLE)
@@ -470,12 +478,17 @@
       for packagename, filename, apk_version_code, store_version_code in list_apks_to_update :
         message += u"\n%s Version : %s -> %s" % (filename ,apk_version_code, store_version_code)
         list_of_packages_to_download.append([packagename, filename])
-      message += "\n\nDo you agree?"
-      dlg = wx.MessageDialog(self, message, 'Updating APKs', wx.ICON_INFORMATION|wx.YES_NO )
-      return_value = dlg.ShowModal()
-      dlg.Destroy()
-
+      print self.autoUpdate
+      if self.autoUpdate:
+        return_value = wx.ID_YES
+      else: 
+        message += "\n\nDo you agree?"
+        dlg = wx.MessageDialog(self, message, 'Updating APKs', wx.ICON_INFORMATION|wx.YES_NO )
+        return_value = dlg.ShowModal()
+        dlg.Destroy()
+      
       if return_value == wx.ID_YES :
+        print "Updating APKs"
         #Progress Dialog
         dlg = wx.ProgressDialog("Updating APKs",
                                  " " * 30 + "\n"*4,
@@ -484,6 +497,9 @@
                                  style = wx.PD_CAN_ABORT|wx.PD_AUTO_HIDE
                                  )
         thread.start_new_thread(download_selection, (self.playstore_api, list_of_packages_to_download, dlg, self.after_download))
+    else:
+        self.autoUpdate = False
+        quit()
 
 class ConfigDialog(wx.Dialog):
   def __init__(self, parent):
@@ -635,6 +651,9 @@
 
     #Connection
     self.panel.connect_to_googleplay_api()
+    if sys.argv[1] == '--update':
+        self.panel.autoUpdate = True 
+        self.panel.prepare_analyse_apks(self.panel)
 
 class App(wx.App):
   def OnInit(self):
--- googleplaydownloader/__init__.py	2015-01-27 10:25:26.000000000 +1100
+++ googleplaydownloader/__init__.py	2015-05-05 10:53:59.000000000 +1000
@@ -4,7 +4,7 @@
 
 import os
 HERE = os.path.abspath(os.path.dirname(__file__))
-with open(os.path.join(HERE, "googleplaydownloader", "version.txt"), "r") as f:
+with open(os.path.join(HERE, "version.txt"), "r") as f:
   __version__ = f.read().strip()
 
 from .googleplaydownloader import main as start_gui
