--- a/lib/tvdb_api.py	2020-04-09 14:26:58.099829873 +1000
+++ b/lib/tvdb_api.py	2020-04-09 14:27:41.737884219 +1000
@@ -1003,6 +1003,14 @@
                 seas_no = int(float(elem_seasnum))
                 ep_no = int(float(elem_epno))
 
+                # Fixes American Dad! issue from TVDB
+                if sid == 73141:
+                    if seas_no==2:
+                        seas_no = 1
+                        ep_no = ep_no+7
+                    if seas_no>2:
+                        seas_no=seas_no-1
+
                 for k, v in iteritems(cur_ep):
                     k = k.lower()
 
