--- src/makefile	2013-07-13 20:16:07.000000000 +1000
+++ src/makefile	2013-07-29 10:10:27.967902520 +1000
@@ -7,7 +7,7 @@
 CFLAGS = --std=c99
 
 all:
-	valac -X -D'GETTEXT_PACKAGE="conky-manager"' --Xcc="-lm" "Main.vala" "MainWindow.vala" "Utility.vala" --pkg glib-2.0 --pkg gio-unix-2.0 --pkg posix --pkg gtk+-3.0 --pkg gee-1.0 -o conky-manager 
+	valac-0.18 -X -D'GETTEXT_PACKAGE="conky-manager"' --Xcc="-lm" "Main.vala" "MainWindow.vala" "Utility.vala" --pkg glib-2.0 --pkg gio-unix-2.0 --pkg posix --pkg gtk+-3.0 --pkg gee-1.0 -o conky-manager 
 
 clean:
 	rm -rf *.o conky-manager
