--- a/sabnzbd/config.py	2017-04-09 20:21:19.000000000 +1000
+++ b/sabnzbd/config.py	2017-04-25 12:25:50.352229239 +1000
@@ -26,7 +26,7 @@
 import shutil
 import sabnzbd.misc
 from sabnzbd.constants import CONFIG_VERSION, NORMAL_PRIORITY, DEFAULT_PRIORITY, MAX_WIN_DFOLDER
-from sabnzbd.utils import configobj
+import configobj 
 from sabnzbd.decorators import synchronized
 
 CONFIG_LOCK = threading.Lock()
