diff -uNar a/sabnzbd/cfg.py b/sabnzbd/cfg.py
--- a/sabnzbd/cfg.py	2017-08-17 18:35:45.000000000 +1000
+++ b/sabnzbd/cfg.py	2017-08-20 08:35:05.057913939 +1000
@@ -322,7 +322,7 @@
 acenter_prio_other = OptionBool('acenter', 'acenter_prio_other', False)
 
 # [ntfosd]
-ntfosd_enable = OptionBool('ntfosd', 'ntfosd_enable', not sabnzbd.WIN32 and not sabnzbd.DARWIN)
+ntfosd_enable = OptionBool('ntfosd', 'ntfosd_enable', False)
 ntfosd_cats = OptionList('ntfosd', 'ntfosd_cats', ['*'])
 ntfosd_prio_startup = OptionBool('ntfosd', 'ntfosd_prio_startup', True)
 ntfosd_prio_download = OptionBool('ntfosd', 'ntfosd_prio_download', False)
