diff -uNar a/sabnzbd/assembler.py b/sabnzbd/assembler.py
--- a/sabnzbd/assembler.py	2017-08-17 18:35:45.000000000 +1000
+++ b/sabnzbd/assembler.py	2017-08-20 08:36:15.616664033 +1000
@@ -36,7 +36,7 @@
 from sabnzbd.articlecache import ArticleCache
 from sabnzbd.postproc import PostProcessor
 import sabnzbd.downloader
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.encoding import unicoder, is_utf8
 from sabnzbd.rating import Rating
 
diff -uNar a/sabnzbd/dirscanner.py b/sabnzbd/dirscanner.py
--- a/sabnzbd/dirscanner.py	2017-08-17 18:35:45.000000000 +1000
+++ b/sabnzbd/dirscanner.py	2017-08-20 08:36:25.401629371 +1000
@@ -29,7 +29,7 @@
 
 import sabnzbd
 from sabnzbd.constants import SCAN_FILE_NAME, VALID_ARCHIVES
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.encoding import platform_encode
 from sabnzbd.newsunpack import is_sevenfile, SevenZip
 import sabnzbd.nzbstuff as nzbstuff
diff -uNar a/sabnzbd/newsunpack.py b/sabnzbd/newsunpack.py
--- a/sabnzbd/newsunpack.py	2017-08-17 18:35:45.000000000 +1000
+++ b/sabnzbd/newsunpack.py	2017-08-20 08:36:37.669585913 +1000
@@ -30,7 +30,7 @@
 
 import sabnzbd
 from sabnzbd.encoding import TRANS, UNTRANS, unicoder, platform_encode, deunicode
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.misc import format_time_string, find_on_path, make_script_path, int_conv, \
     real_path, globber, globber_full, get_all_passwords, renamer, clip_path, \
     has_win_device, calc_age, long_path
diff -uNar a/sabnzbd/postproc.py b/sabnzbd/postproc.py
--- a/sabnzbd/postproc.py	2017-08-17 18:35:45.000000000 +1000
+++ b/sabnzbd/postproc.py	2017-08-20 08:36:48.738546701 +1000
@@ -49,7 +49,7 @@
 import sabnzbd.nzbqueue
 import sabnzbd.database as database
 import sabnzbd.notifier as notifier
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 import sabnzbd.utils.checkdir
 
 # Match samples
