diff -uar a/sabnzbd/assembler.py b/sabnzbd/assembler.py
--- a/sabnzbd/assembler.py	2017-04-09 20:21:18.000000000 +1000
+++ b/sabnzbd/assembler.py	2017-04-25 12:36:49.882919423 +1000
@@ -36,7 +36,7 @@
 from sabnzbd.articlecache import ArticleCache
 from sabnzbd.postproc import PostProcessor
 import sabnzbd.downloader
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.encoding import unicoder, deunicode, is_utf8
 from sabnzbd.rating import Rating
 
diff -uar a/sabnzbd/dirscanner.py b/sabnzbd/dirscanner.py
--- a/sabnzbd/dirscanner.py	2017-04-09 20:21:19.000000000 +1000
+++ b/sabnzbd/dirscanner.py	2017-04-25 12:37:02.528874788 +1000
@@ -29,7 +29,7 @@
 
 import sabnzbd
 from sabnzbd.constants import SCAN_FILE_NAME, VALID_ARCHIVES
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.encoding import platform_encode
 from sabnzbd.newsunpack import is_sevenfile, SevenZip
 import sabnzbd.nzbstuff as nzbstuff
diff -uar a/sabnzbd/newsunpack.py b/sabnzbd/newsunpack.py
--- a/sabnzbd/newsunpack.py	2017-04-09 20:21:19.000000000 +1000
+++ b/sabnzbd/newsunpack.py	2017-04-25 12:36:56.474896157 +1000
@@ -31,7 +31,7 @@
 import sabnzbd
 from sabnzbd.encoding import TRANS, UNTRANS, unicode2local, \
     reliable_unpack_names, unicoder, platform_encode, deunicode
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 from sabnzbd.misc import format_time_string, find_on_path, make_script_path, int_conv, \
     flag_file, real_path, globber, globber_full, get_all_passwords, renamer, clip_path, \
     has_win_device, calc_age
diff -uar a/sabnzbd/postproc.py b/sabnzbd/postproc.py
--- a/sabnzbd/postproc.py	2017-04-09 20:21:20.000000000 +1000
+++ b/sabnzbd/postproc.py	2017-04-25 12:37:07.411857550 +1000
@@ -49,7 +49,7 @@
 import sabnzbd.nzbqueue
 import sabnzbd.database as database
 import sabnzbd.notifier as notifier
-import sabnzbd.utils.rarfile as rarfile
+import rarfile
 import sabnzbd.utils.checkdir
 
 
