--- dex	2013-01-01 20:20:45.417859436 +1100
+++ dex	2013-01-01 20:19:59.312435833 +1100
@@ -506,14 +506,18 @@
 		"""
 		_exec = True
 		_try = self.TryExec
+		execFile = os.path.split(self.Exec.split(' ')[0])[1]
 		if _try and not (os.path.isabs(_try) and os.path.isfile(_try)) and not which(_try):
 			_exec = False
 
 		if _exec:
 			path = self.Path
 			cmd = self._build_cmd(self.Exec)
+			_pid = subprocess.getoutput('pgrep -u $USERNAME -x '+execFile) 
 			if not cmd:
 				raise ApplicationExecException('Failed to build command string.')
+			if _pid:
+				raise ApplicationExecException('Application already running with pid ' + _pid)
 			if dryrun or verbose:
 				if verbose:
 					print('Autostart file: %s' % self.filename)

