diff -uNar ../spotweb-o/lib/dbeng/dbeng_mysql.php ./lib/dbeng/dbeng_mysql.php
--- ../spotweb-o/lib/dbeng/dbeng_mysql.php	2013-09-23 15:35:41.699594148 +1000
+++ ./lib/dbeng/dbeng_mysql.php	2013-09-23 15:40:41.079609902 +1000
@@ -24,14 +24,14 @@
 	}
 	
 	function connect() {
-		$this->_conn = mysql_connect($this->_db_host, $this->_db_user, $this->_db_pass);
+		$this->_conn = ($GLOBALS["___mysqli_ston"] = mysqli_connect($this->_db_host,  $this->_db_user,  $this->_db_pass));
 		
 		if (!$this->_conn) {
-			throw new DatabaseConnectionException("Unable to connect to MySQL server: " . mysql_error());
+			throw new DatabaseConnectionException("Unable to connect to MySQL server: " . ((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_error($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)));
 		} # if 
 				
-		if (!@mysql_select_db($this->_db_db, $this->_conn)) {
-			throw new DatabaseConnectionException("Unable to select MySQL db: " . mysql_error($this->_conn));
+		if (!@((bool)mysqli_query( $this->_conn, "USE $this->_db_db"))) {
+			throw new DatabaseConnectionException("Unable to select MySQL db: " . ((is_object($this->_conn)) ? mysqli_error($this->_conn) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)));
 			return false;
 		} # if
 		
@@ -40,14 +40,14 @@
     } # connect()
 		
 	function safe($s) {
-		return mysql_real_escape_string($s);
+		return ((isset($GLOBALS["___mysqli_ston"]) && is_object($GLOBALS["___mysqli_ston"])) ? mysqli_real_escape_string($GLOBALS["___mysqli_ston"], $s) : ((trigger_error("[MySQLConverterToo] Fix the mysql_escape_string() call! This code does not work.", E_USER_ERROR)) ? "" : ""));
 	} # safe
 
 	function rawExec($s) {
 		SpotTiming::start(__FUNCTION__);
-		$tmpRes = mysql_unbuffered_query($s, $this->_conn);
+		$tmpRes = mysqli_query( $this->_conn, $s, MYSQLI_USE_RESULT);
 		if ($tmpRes === false) {
-			throw new SqlErrorException(mysql_error($this->_conn), mysql_errno($this->_conn));
+			throw new SqlErrorException(((is_object($this->_conn)) ? mysqli_error($this->_conn) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)), ((is_object($this->_conn)) ? mysqli_errno($this->_conn) : (($___mysqli_res = mysqli_connect_errno()) ? $___mysqli_res : false)));
 		} # if
 		SpotTiming::stop(__FUNCTION__, array($s));
 		
@@ -62,7 +62,7 @@
 
 		$res = $this->exec($s, $p);
 		if (!is_bool($res)) {
-			mysql_free_result($res);
+			((mysqli_free_result($res) || (is_object($res) && (get_class($res) == "mysqli_result"))) ? true : false);
 		} # if
 		
 		SpotTiming::stop(__FUNCTION__, array($s,$p));
@@ -73,8 +73,8 @@
 		SpotTiming::start(__FUNCTION__);
 		
 		$res = $this->exec($s, $p);
-		$row = mysql_fetch_array($res);
-		mysql_free_result($res);
+		$row = mysqli_fetch_array($res);
+		((mysqli_free_result($res) || (is_object($res) && (get_class($res) == "mysqli_result"))) ? true : false);
 		
 		SpotTiming::stop(__FUNCTION__, array($s,$p));
 		
@@ -86,12 +86,12 @@
 		$rows = array();
 
 		$res = $this->exec($s, $p); 
-		while ($rows[] = mysql_fetch_assoc($res));
+		while ($rows[] = mysqli_fetch_assoc($res));
 
 		# remove last element (false element)
 		array_pop($rows); 
 		
-		mysql_free_result($res);
+		((mysqli_free_result($res) || (is_object($res) && (get_class($res) == "mysqli_result"))) ? true : false);
 		SpotTiming::stop(__FUNCTION__, array($s,$p));
 		
 		return $rows;
@@ -137,7 +137,7 @@
 	 *    http://nl.php.net/manual/en/function.mysql-info.php#36008
 	 */
 	function get_mysql_info() {
-		$strInfo = mysql_info($this->_conn);
+		$strInfo = mysqli_info($this->_conn);
 	   
 		$return = array();
 		preg_match("/Rows matched: ([0-9]*)/", $strInfo, $rows_matched);
@@ -153,7 +153,7 @@
 	} # rows()
 	
 	function lastInsertId($tableName) {
-		return mysql_insert_id($this->_conn);
+		return ((is_null($___mysqli_res = mysqli_insert_id($this->_conn))) ? false : $___mysqli_res);
 	} # lastInsertId
 
 	
diff -uNar ../spotweb-o/tests/testInsert.php ./tests/testInsert.php
--- ../spotweb-o/tests/testInsert.php	2013-09-23 15:35:41.709594148 +1000
+++ ./tests/testInsert.php	2013-09-23 15:40:42.327609968 +1000
@@ -1,9 +1,9 @@
 <?php
 error_reporting(E_ALL);	
 
- mysql_connect('localhost', 'spotweb', 'spotweb', 'spotweb');
+ ($GLOBALS["___mysqli_ston"] = mysqli_connect('localhost',  'spotweb',  'spotweb'));
 
-mysql_select_db('spotweb');
+((bool)mysqli_query($GLOBALS["___mysqli_ston"], "USE spotweb"));
 
   $data = array('5412321e9b3024b668854ef1ffff606a140@free.pt', 'testposter', 'Test Title', 'tagtagtagtagtag', 0, 'a10|', 'b3|', 'c6|', 'd11|', 'z1|', time(), -time(), 0, NULL, 0, 0, 0, '');
 
@@ -20,9 +20,9 @@
   } # for
   $sql = substr($sql, 0, -1);
 
-  	 mysql_query($sql);
+  	 mysqli_query($GLOBALS["___mysqli_ston"], $sql);
 
-  	 if (mysql_errno() != 0) {
+  	 if (((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_errno($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_errno()) ? $___mysqli_res : false)) != 0) {
   	 	echo "ffs!" . PHP_EOL;
   	 	break;
   	 } # if
@@ -32,4 +32,4 @@
 
    echo "Total time for 500k inserts: " . ($y - $x) . PHP_EOL;
 
-die(mysql_error().PHP_EOL);
+die(((is_object($GLOBALS["___mysqli_ston"])) ? mysqli_error($GLOBALS["___mysqli_ston"]) : (($___mysqli_res = mysqli_connect_error()) ? $___mysqli_res : false)).PHP_EOL);
