diff -uNar intranet/comingseasons.php www/localhost/htdocs/intranet/comingseasons.php
--- intranet/comingseasons.php	2013-10-11 15:50:28.552147908 +1100
+++ www/localhost/htdocs/intranet/comingseasons.php	2013-10-11 15:48:28.878151497 +1100
@@ -3,10 +3,20 @@
 // Include Settings
 include("serverconfig.php");
 
+function getURL ($url,$port) {
+    $host = explode('/',$url,2)[0];
+    $weburl = $host.":".$port;
+    if ($host <> $url) {
+        $root = explode('/',$url,2)[1];
+        $weburl = $weburl."/".$root;
+    }
+    return $weburl;
+}
+
 // Edit Settings
 $username = $config['sickbeardUsername'];
 $password = $config['sickbeardPassword'];
-$ip = $config['sickbeardURL'].":".$config['sickbeardPort'];
+$ip = getURL($config['sickbeardURL'],$config['sickbeardPort']);
 $api = $config['sickbeardAPI'];
 // End Settings
 
@@ -31,15 +41,15 @@
 			<?php
 			echo "<ul>";
 			// Run through each feed item
-			foreach($sbJSON->{data}->{later} as $show) {
+			foreach($sbJSON->{'data'}->{'later'} as $show) {
 				// Only grab shows of episode 1
-				if($show->{episode} == "1") {
+				if($show->{'episode'} == "1") {
 					// Reformat date
-					$newDate = date("l, j F Y", strtotime($show->{airdate}));
+					$newDate = date("l, j F Y", strtotime($show->{'airdate'}));
 					// Show Details
 					echo "<li>";
 					echo "<img src='http://".$ip."/showPoster/?show=".$show->{'tvdbid'}."&which=banner' class='showbanner' />";
-					echo "<strong>".$show->{show_name} . "</strong><br />Season " . $show->{season} . "<br /><br />Starts " .$newDate;
+					echo "<strong>".$show->{'show_name'} . "</strong><br />Season " . $show->{'season'} . "<br /><br />Starts " .$newDate;
 					echo "</li>";
 				}
 			}
@@ -49,4 +59,4 @@
 		<script src="js/jquery.js"></script>
 		<script src="js/scripts.js"></script>
 	</body>
-</html>
\ No newline at end of file
+</html>
diff -uNar intranet/lib/functions.php www/localhost/htdocs/intranet/lib/functions.php
--- intranet/lib/functions.php	2013-10-11 15:50:28.558147908 +1100
+++ www/localhost/htdocs/intranet/lib/functions.php	2013-10-11 14:55:16.644247244 +1100
@@ -7,36 +7,48 @@
 
 ================================================================== */
 
+function getURL ($url,$port) {
+    $host = explode('/',$url,2)[0];
+    $weburl = $host.":".$port;
+    if ($host <> $url) {
+    	$root = explode('/',$url,2)[1];
+	$weburl = $weburl."/".$root;
+    }
+    return $weburl;
+}
     # Sickbeard URL Joining
     if($config['sickbeardHTTPS']) {$sickbeardProtocol = "https";} else {$sickbeardProtocol = "http";}
     if($config['sickbeardUsername']) {
-        $sickbeardURL = $sickbeardProtocol."://".$config['sickbeardUsername'].":".$config['sickbeardPassword']."@".$config['sickbeardURL'].":".$config['sickbeardPort'];
+        $sickbeardURL = $sickbeardProtocol."://".$config['sickbeardUsername'].":".$config['sickbeardPassword']."@";
     } else {
-        $sickbeardURL = $sickbeardProtocol."://".$config['sickbeardURL'].":".$config['sickbeardPort'];
+        $sickbeardURL = $sickbeardProtocol."://";
     }
+    $sickbeardURL = $sickbeardURL.getURL($config['sickbeardURL'],$config['sickbeardPort']);
 
     # Sabnzbd URL Joining
     if($config['sabnzbdHTTPS']) {$sabProtocol = "https";} else {$sabProtocol = "http";}
     if($config['sabnzbdUsername']) {
-        $sabURL = $sabProtocol."://".$config['sabnzbdUsername'].":".$config['sabnzbdPassword']."@".$config['sabnzbdURL'].":".$config['sabnzbdPort'];
+        $sabURL = $sabProtocol."://".$config['sabnzbdUsername'].":".$config['sabnzbdPassword']."@";
     } else {
-        $sabURL = $sabProtocol."://".$config['sabnzbdURL'].":".$config['sabnzbdPort'];
+        $sabURL = $sabProtocol."://";
     }
+    $sabURL = $sabURL.getURL($config['sabnzbdURL'],$config['sabnzbdPort']);
 
     # Transmission Joining
     if($config['transmissionUsername']) {
-        $transmissionURL = "http://".$config['transmissionURL'].":".$config['transmissionPort']."@".$config['transmissionUsername'].":".$config['transmissionPassword'];
+        $transmissionURL = "http://".$config['transmissionUsername'].":".$config['transmissionPassword']."@";
     } else {
-        $transmissionURL = "http://".$config['transmissionURL'].":".$config['transmissionPort'];
+        $transmissionURL = "http://";
     }
+    $transmissionURL = $transmissionURL.getURL($config['transmissionURL'],$config['transmissionPort']);
 
     # Headphones Joining
     if($config['headphonesHTTPS']) {$headphonesProtocol = "https";} else {$headphonesProtocol = "http";}
-    $headphonesURL = $headphonesProtocol."://".$config['headphonesURL'].":".$config['headphonesPort'];
+    $headphonesURL = $headphonesProtocol."://".getURL($config['headphonesURL'],$config['headphonesPort']);
 
     # Couchpotato Joining
     if($config['couchpotatoHTTPS']) {$couchpotatoProtocol = "https";} else {$couchpotatoProtocol = "http";}
-    $couchpotatoURL = $couchpotatoProtocol."://".$config['couchpotatoURL'].":".$config['couchpotatoPort'];
+    $couchpotatoURL = $couchpotatoProtocol."://".getURL($config['couchpotatoURL'],$config['couchpotatoPort']);
 
 /* ==================================================================
 
@@ -69,4 +81,4 @@
     return $size; 
     } 
 
-?>
\ No newline at end of file
+?>
--- index.php	2013-10-11 15:50:28.552147908 +1100
+++ www/localhost/htdocs/index.php	2013-10-11 15:40:54.360165130 +1100
@@ -76,9 +76,9 @@
 					if (substr($episode->date,0,10) == $todaysDate && $episode->status == "Snatched") :
 
 						// Check Quality Snatched
-						if ($episode->{quality} == "SD TV") :
+						if ($episode->{'quality'} == "SD TV") :
 							$quality = "sd";
-						elseif ($episode->{quality} == "HD TV") :
+						elseif ($episode->{'quality'} == "HD TV") :
 							$quality = "hd";
 						endif;
 
@@ -125,7 +125,7 @@
 				<a href="#" class="go actionButton small">&gt;</a>
 				<?php
 
-					$sabStatusXML = $sabURL."/sabnzbd/api?mode=qstatus&output=xml&apikey=".$config['sabnzbdAPI'];
+					$sabStatusXML = $sabURL."/api?mode=qstatus&output=xml&apikey=".$config['sabnzbdAPI'];
 					if($config['debug']){echo "SABnzbd Status URL: ".$sabStatusXML;}
 					$data = simplexml_load_file($sabStatusXML);
 					$filename = $data->jobs[0]->job->filename;
@@ -158,7 +158,7 @@
 				<a href="#" class="go actionButton small">&lt;</a>
 				<?php
 
-					$data = simplexml_load_file($sabURL."/sabnzbd/api?mode=history&start=0&limit=5&output=xml&apikey=".$config['sabnzbdAPI']);
+					$data = simplexml_load_file($sabURL."/api?mode=history&start=0&limit=5&output=xml&apikey=".$config['sabnzbdAPI']);
 					echo "<ul>";
 					foreach($data->slots[0] as $slot) {
 						echo "<li>".$slot->category." - ".$slot->nzb_name."</li>";
@@ -289,4 +289,4 @@
 		<script src="intranet/js/jquery.js"></script>
 		<script src="intranet/js/scripts.js"></script>
 	</body>
-</html>
\ No newline at end of file
+</html>

