--- MANIFEST.in	2020-01-07 14:51:50.735446468 +1100
+++ MANIFEST.in	2020-01-07 14:52:01.698218800 +1100
@@ -24,22 +24,6 @@
 include fdroid
 include gradlew-fdroid
 include LICENSE
-include locale/bo/LC_MESSAGES/fdroidserver.mo
-include locale/de/LC_MESSAGES/fdroidserver.mo
-include locale/es/LC_MESSAGES/fdroidserver.mo
-include locale/fr/LC_MESSAGES/fdroidserver.mo
-include locale/hu/LC_MESSAGES/fdroidserver.mo
-include locale/it/LC_MESSAGES/fdroidserver.mo
-include locale/ko/LC_MESSAGES/fdroidserver.mo
-include locale/nb_NO/LC_MESSAGES/fdroidserver.mo
-include locale/pl/LC_MESSAGES/fdroidserver.mo
-include locale/pt_BR/LC_MESSAGES/fdroidserver.mo
-include locale/pt_PT/LC_MESSAGES/fdroidserver.mo
-include locale/ru/LC_MESSAGES/fdroidserver.mo
-include locale/tr/LC_MESSAGES/fdroidserver.mo
-include locale/uk/LC_MESSAGES/fdroidserver.mo
-include locale/zh_Hans/LC_MESSAGES/fdroidserver.mo
-include locale/zh_Hant/LC_MESSAGES/fdroidserver.mo
 include makebuildserver
 include README.md
 include tests/androguard_test.py
