diff -uNar ./html/main.php PHPCoin-99999-new/html/main.php
--- ./html/main.php	2013-07-19 14:26:10.117714720 +1000
+++ PHPCoin-99999-new/html/main.php	2013-07-19 14:27:50.440846961 +1000
@@ -24,7 +24,8 @@
     <label>Name</label> <?php echo $_SESSION['name'];?>
 </div>
 <?php
-    $actAcount = end(explode("_",$_SESSION['btaccount']));
+    $tmp = explode("_",$_SESSION['btaccount']);
+    $actAcount = end($tmp);
     $accountBalance = 0;
 ?>
 <div class="infoLine">
diff -uNar ./login/login.php PHPCoin-99999-new/login/login.php
--- ./login/login.php	2013-07-19 14:26:10.119715720 +1000
+++ PHPCoin-99999-new/login/login.php	2013-07-19 14:27:17.772522348 +1000
@@ -11,7 +11,7 @@
 </head>
 <body>
 <?php
-    if($error){
+    if(isset($error) && ($error)) {
 ?>
   <div class="error"><?php echo $error;?></div>
 <?php        
@@ -119,4 +119,4 @@
 ?>    
    </div>
 </body>
-</html>
\ No newline at end of file
+</html>
diff -uNar ./proc/doCreateAccount.php PHPCoin-99999-new/proc/doCreateAccount.php
--- ./proc/doCreateAccount.php	2013-07-19 14:26:10.119715720 +1000
+++ PHPCoin-99999-new/proc/doCreateAccount.php	2013-07-19 14:28:04.196720876 +1000
@@ -19,7 +19,7 @@
     if($fwd == 1 && !$fwdto) $e[] = "You must enter a bitcoin address to forward to!";
     
     if(empty($e)){
-        $sql = "SELECT * FROM accounts WHERE account_name LIKE '$name' AND uid = {$_SESSION['id']} AND id != $aid";
+        $sql = "SELECT * FROM accounts WHERE account_name LIKE '$name' AND uid = {$_SESSION['id']}"; ## AND id != $aid";
         $q = mysql_query($sql);
         if(mysql_num_rows($q)) $e[] = "You already have another account with that same name!";                
     }    
@@ -47,4 +47,4 @@
     }else{
         $error = implode("<br/>",$e);
     }    
-?>
\ No newline at end of file
+?>
