--- www/automated.config.php	2014-05-19 14:51:16.920026170 +1000
+++ www/automated.config.php	2014-05-19 14:51:16.920026170 +1000
@@ -70,7 +70,7 @@
 define('DS', DIRECTORY_SEPARATOR);
 
 // These are file path constants
-define('nZEDb_ROOT', realpath(dirname(dirname(__FILE__))) . DS);
+define('nZEDb_ROOT', '/usr/share/webapps/nzedb/9999/hostroot/nzedb-9999'. DS);
 
 // Used to refer to the main lib class files.
 define('nZEDb_LIB', nZEDb_ROOT . 'nzedb' . DS);
@@ -83,7 +83,7 @@
 define('nZEDb_MISC', nZEDb_ROOT . 'misc' . DS);
 
 // Refers to the web root for the Smarty lib
-define('nZEDb_WWW', nZEDb_ROOT . 'www' . DS);
+define('nZEDb_WWW',  '/usr/share/webapps/nzedb/9999/htdocs' . DS);
 
 // Used to refer to the resources folder
 define('nZEDb_RES', nZEDb_ROOT . 'resources' . DS);
